docs/nos-client/api.md

```diff
@@ -5,17 +5,60 @@ sidebar_label: API Overview
 ---
 
 **NOTE: The API is in active development and may change frequently.  Be sure to check back
-regularly. All API functions will have a maximum of ONE argument: a JSON object containing all the necessary info.**
+regularly. All API functions will have a maximum of ONE argument: a JSON object containing all the
+necessary info.**
 
 The nOS client provides a prebuilt API that is still growing.  For any dApp running through the
-client, the API can be accessed via `window.NOS.V1`.  All functions return a
-[`Promise`](https://www.google.com/search?q=js+promise&oq=js+promise&aqs=chrome..69i57j69i60l3.731j0j7&sourceid=chrome&ie=UTF-8), which can be used to determine if the call succeeded or failed.  Some calls will fail if the
-user rejects the action.
```

Comment:
To clarify this removal, I moved this part down to the new "Functions" heading since it pertains to that.